recipe/meta.yaml

```diff
@@ -8,24 +8,24 @@ source:
   sha256: eb4b9c2956e38f89d588a0d0af0e27a03f1e1d2be705a828bbd12cfba0b020fa
 
 build:
-  number: 0 
+  number: 1
   script: pip install --no-deps .
 
 requirements:
   build:
     - python
     - pip
     - zlib 1.2*
-    - libtiff 4*
-    - jpeg 8*
+    - libtiff 4.0.6
+    - jpeg 9*
     - freetype 2.5*
     - tk
   run:
     - python
-    - jpeg 8*
+    - jpeg 9*
     - zlib 1.2*
     - freetype 2.5*
-    - libtiff 4*
+    - libtiff 4.0.6
```

Comment:
> I suggest avoiding that - it is work that we will only need to undo eventually - we should try to find a long-term solution.

I would like to avoid it, but I cannot see a way forward without re-building Python 3.4 and 3.5 with the latest `xz`.

> I don't understand that... the defaults libtiff looks like it is pinned properly, and shouldn't allow a jpeg 8d installation...

The current setup we have is broken as people cannot mix conda-forge binaries that are pinned against `jpeg 8d` (like pillow) and `jpeg 9b` (like `gdal` and all its dependencies.)

This change gets us consistent within `conda-forge` and I believe better prepared to update once we have Python 3.4 and 3.5 in the channel.